### DISCUSSION TITLE
browse field with elfinder

### DISCUSSION THREAD

**Discussion by soluzione-software** [Laravel-Backpack/community-forum#1061] [Q&A (Help)]
I try to use browse field
CRUD::field([   // Browse
    'name'  => 'image',
    'label' => 'Image',
    'type'  => 'browse'
]);

my backpack show the public folder
i correctly installed the driver

the problem is this:
if in the elfinder.php config file i put a folder in the dir key it works 
/*
    |--------------------------------------------------------------------------
    | Upload dir
    |--------------------------------------------------------------------------
    |
    | The dir where to store the images (relative from public).
    |
    */
    'dir' => ['uploads' ],

if i put a disk in the disk key it doesn't works
 /*
    |--------------------------------------------------------------------------
    | Filesystem disks (Flysytem)
    |--------------------------------------------------------------------------
    |
    | Define an array of Filesystem disks, which use Flysystem.
    | You can set extra options, example:
    |
    | 'my-disk' => [
    |        'URL' => url('to/disk'),
    |        'alias' => 'Local storage',
    |    ]
    */
    'disks' => [
        's3',

    ],

why? can anyone help me please?

**Comment by pxpm** [comment]
Hello @soluzione-software what does not work ? 

What's the error ? Can you share a screenshot? Does your `s3` disk is properly configured and working outside of elfinder ? 

Check if anything of this helps:
https://github.com/barryvdh/laravel-elfinder/issues/188
https://github.com/barryvdh/laravel-elfinder/issues/236#issuecomment-422964351

It's possible that elfinder does not play well with s3 buckets, but I don't know as I don't use it with cloud disks. 

Let us know if you find something 🙏 

Cheers

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
